src/utils/credentials.ts

```diff
@@ -88,13 +88,13 @@ export class Credentials {
     return Credentials.enumToUsersMap.get(userRole)
   }
 
-  private static checkPermission(
-    userPermission: Permissions | undefined,
-    demandedPermission: Permissions
-  ) {
-    if (!userPermission)
-      throw new Error('Unauthorized access. User permission is not valid')
-    if (userPermission < demandedPermission)
-      throw new Error('Unauthorized access')
-  }
+  // private static checkPermission(
+  //   userPermission: Permissions | undefined,
+  //   demandedPermission: Permissions
+  // ) {
+  //   if (!userPermission)
+  //     throw new Error('Unauthorized access. User permission is not valid')
+  //   if (userPermission < demandedPermission)
+  //     throw new Error('Unauthorized access')
+  // }
```

Comment:
Przyda się, czy do usunięcia?

Reply:
Przyda się. Webstorm coś mi trochę krzyczał, to zakomentowałem. To jest nieużywany, bo te role jakby nie działają - wyłączyliśmy je